pubsub-mapped-api/src/main/java/com/google/pubsub/clients/consumer/ack/MessageDispatcher.java

```diff
@@ -76,9 +75,9 @@
   private final FlowController flowController;
   private final MessageWaiter messagesWaiter;
 
+  private Set<String> pendingNacks;
```

Comment:
it would be nice to say here explicitly that these are concurrent (thread safe).

Reply:
Okay.